native/src/components/MapView.tsx

```diff
@@ -28,16 +39,13 @@ import MapAttribution from './MapsAttribution'
 import Icon from './base/Icon'
 import IconButton from './base/IconButton'
 
-// Has to be set even if we use map libre
-MapLibreGL.setAccessToken(null)
-
 const MapContainer = styled.View`
   flex: 1;
   flex-direction: row;
   justify-content: center;
 `
 
-const StyledMap = styled(MapLibreGL.MapView)`
+const StyledMap = styled(MLMapView)`
```

Comment:
🙃 I'd prefer `MapLibreMapView` as `ML` is usually machine learning... which really confused me for a bit. Generally we try to avoid abbreviations.
```suggestion
const StyledMap = styled(MLMapView)`
```